src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -796,6 +806,32 @@ def process_string_input(part_name, parsed_question, data_dict):
 
     return replace_tags(html)
 
+
+def process_workspace(part_name, parsed_question, data_dict):
+    """Processes markdown format of workspace questions and returns PL HTML
+    Args:
+        part_name (string): Name of the question part being processed (e.g., part1, part2, etc...)
+        parsed_question (dict): Dictionary of the MD-parsed question (output of `read_md_problem`)
+        data_dict (dict): Dictionary of the `data` dict created after running server.py using `exec()`
+
+    Returns:
+        html: A string of HTML that is part of the final PL question.html file.
+    """
+    if "pl-customizations" in parsed_question["header"][part_name]:
+        if len(parsed_question["header"][part_name]["pl-customizations"]) > 0:
+            raise ValueError("pl-customizations are not supported for workspace questions")
+
+
+    html = f"""<pl-question-panel>\n<markdown>{parsed_question['body_parts_split'][part_name]['content']}</markdown>\n</pl-question-panel>\n\n"""
+
+    html += f"""<pl-workspace></pl-workspace>"""
+
+    if parsed_question["header"][part_name].get("gradingMethod", None) == "External":
+        html += f"""<pl-submission-panel>\n\t<pl-external-grader-results></pl-external-grader-results>\n\t<pl-file-preview></pl-file-preview></pl-submission-panel>"""
```

Comment:
Based off of [PrairieLearn's docs](https://prairielearn.readthedocs.io/en/latest/workspaces/#infojson-for-ungraded-workspace), can remove if not needed

Reply:
I think this is needed.